calico_cni_test.go

```diff
@@ -390,63 +390,4 @@ var _ = Describe("CalicoCni", func() {
 			})
 		})
 	})
-
-	Describe("CNI should be resilient to an already existing route", func() {
-		Context("add a new container when the route already exists  ", func() {
-			netconf := fmt.Sprintf(`
-			{
-			  "cniVersion": "%s",
-			  "name": "net1",
-			  "type": "calico",
-			  "etcd_endpoints": "http://%s:2379",
-			  "ipam": {
-			    "type": "host-local",
-			    "subnet": "10.0.0.0/24"
-			  }
-			}`, cniVersion, os.Getenv("ETCD_IP"))
-
-			It("should successfully add and network the container", func() {
-				containerIPStr := "10.0.0.55"
-				_, containerIP, _ := net.ParseCIDR("10.0.0.55/32")
-
-				By("Manually programming the route that CNI plugin is going to add.")
-				netlink.RouteAdd(
-					&netlink.Route{
```

Comment:
@caseydavenport I'm removing this whole test (unrelated to this PR) because you can't add a route without the interface being present first which is why this test was failing. There's no easy way to reproduce the interface and route being present before we call CNI ADD, so I'm removing this test for now